assopy/models.py

```diff
@@ -582,8 +582,8 @@ def __str__(self):
 
 
 class VatFare(models.Model):
-    fare = models.ForeignKey('conference.fare')
-    vat = models.ForeignKey(Vat)
+    fare = models.ForeignKey('conference.fare', on_delete=models.CASCADE)
```

Comment:
unless this causes some big circular import I'd opt for using class here directly instead of the string.